osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -27,131 +24,26 @@ public abstract class WorkingBeatmap : IDisposable
         /// </summary>
         public PlayMode? PreferredPlayMode;
 
-        public PlayMode PlayMode => beatmap?.BeatmapInfo?.Mode > PlayMode.Osu ? beatmap.BeatmapInfo.Mode : PreferredPlayMode ?? PlayMode.Osu;
+        public PlayMode PlayMode => Beatmap?.BeatmapInfo?.Mode > PlayMode.Osu ? Beatmap.BeatmapInfo.Mode : PreferredPlayMode ?? PlayMode.Osu;

```

Comment:
This subtly changes the behavior of WorkingBeatmap.PlayMode but I'm pretty sure it isn't a meaningful difference